### PR TITLE
Set Context Invalid in Multi-tenant Db2 #1032

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/FHIRDbDAOImpl.java [hunk 1]
```diff
@@ -247,6 +222,67 @@ public Connection getConnection() throws FHIRPersistenceDBConnectException {
...
+        String tenantKey = null;
+        String dsId = FHIRRequestContext.get().getDataStoreId();
+        boolean multiTenantFeature = false;
+
+        // Retrieve the property group pertaining to the specified datastore.
+        // Find and set the tenantKey for the request, otherwise subsequent pulls from the pool
+        // miss the tenantKey.
+        String dsPropertyName = FHIRConfiguration.PROPERTY_DATASOURCES + "/" + dsId;
+        PropertyGroup dsPG = FHIRConfigHelper.getPropertyGroup(dsPropertyName);
+        if (dsPG != null) {
```
Comment (albertwang-ibm) [inline fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/FHIRDbDAOImpl.java:243]: should we handle dsPG == null situation?

Comment (prb112) [reply fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/FHIRDbDAOImpl.java:243]: let me add a log

Comment (albertwang-ibm) [reply fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/FHIRDbDAOImpl.java:243]: and maybe a exception?

Comment (prb112) [reply fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/FHIRDbDAOImpl.java:243]: fixed, it's a good idea

Comment (albertwang-ibm) [reply fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/FHIRDbDAOImpl.java:243]: cool, thanks!

**Pull request merged**